src/components/trophy-banner/trophy-banner-component.vue

```diff
@@ -0,0 +1,22 @@
+<style lang="scss">
```

Comment:
This doesn't really need to be its own component, you can just have the div with the appropriate css class directly in the `prize-earned` component

Reply:
done!